lib/actions/transform.js

```diff
@@ -8,14 +11,30 @@ const { JsonataTransform } = require('@elastic.io/component-commons-library');
  */
 async function processAction(msg, cfg) {
   let result;
+
+  function manualTransform(self) {
+    const { expression } = cfg;
+    const compiledExpression = jsonata(expression);
+    if (self && self.getFlowVariables) {
+      compiledExpression.assign('getFlowVariables', () => self.getFlowVariables());
+    }
+    compiledExpression.registerFunction('iso2to3', (Iso2) => lookup.byIso(Iso2));
+    compiledExpression.registerFunction('phoneNumtoIso3', (phoneCode) => lookup.byIso(phoneCode));
+    result = compiledExpression.evaluate(msg);
+    self.logger.info(JSON.stringify(result));
+    self.logger.info(JSON.stringify(lookup.byIso(826)));
```

Comment:
Was this just for testing?  It looks like a hardcoded phone prefix.

Reply:
Yep this was just for testing. Removed!